data/card_catalogue/anger.json

```diff
@@ -41,5 +41,41 @@
       "flavour": "All life is precious, even the lives of your enemies",
       "image": "assets/core/card/illustration/Lunge.png"
     }
+  },
+  {
+    "title": "Rue the Day",
+    "upgrade": "",
+    "tier": "green",
+    "details": {
+      "rules": "Name up a character who has wronged you. They feel an remorse for what they have done.",
```

Comment:
This is the most I think we should do for revenge mechanics. They tend to be too niche in most scenarios